meta-lmp-base/classes/uboot-fitimage.bbclass

```diff
@@ -208,7 +221,7 @@ do_deploy_prepend() {
 	else
 		cd ${B}
 		UBOOT_LOAD_ADDR=`grep 'define CONFIG_SYS_TEXT_BASE' u-boot.cfg | cut -d' ' -f 3`
-		uboot_fitimage_assemble ${UBOOT_ITB_BINARY} ${UBOOT_LOAD_ADDR} ${OPTEE_LOAD_ADDR} ${ATF_LOAD_ADDR}
+		uboot_fitimage_assemble ${UBOOT_ITB_BINARY} ${UBOOT_LOAD_ADDR} ${OPTEE_LOAD_ADDR} ${ATF_LOAD_ADDR} ${BOOTSCR_PRELOAD_ADDR}
```

Comment:
Follow the other vars, use BOOTSCR_LOAD_ADDR.

Reply:
@ricardosalveti comment here didn't get addressed.  Was this change rejected?